docs/source/notebooks/clv/clv-quickstart.ipynb

```diff
@@ -0,0 +1,2270 @@
+{
```

Comment:
<p>Shall we use quotations or a style that shoes this is taken exactly as in the lifetimes package?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I copied a lot of the text, not just this. What about saying something like this in the beginning (maybe in an Info box if we have those rendering nicely in our docs): </p><p> </p><p>Large portions of this quickstart are copied directly from lifetimes, including plots and descriptions?</p>